06_neighborhoods/Transportation/transit_trips_cost_city.qmd

```diff
@@ -134,6 +135,52 @@ transit_cost_tracts_2019 <- transport_tracts_2019 %>%
   select(GEOID, state, county, tract, blkgrps, population, households, t_80ami)
 ```
 
+
+### 2020
+
+Bring in all the downloaded CSVs (state-level tracts) & combine them into one nation-wide file for tracts
+
+```{r}
+tracts20files <- list.files(path = here::here("06_neighborhoods",
+                                             "Transportation", 
+                                             "data", 
+                                             "2020_tract"), pattern = "*.csv")
+print(tracts20files)
+
+tractpath20 = file.path(here::here("06_neighborhoods",
+                                             "Transportation", 
+                                             "data", 
+                                             "2020_tract"), tracts20files)
+print(tractpath20)
+
+transport_tracts_2020 <- map_df(tractpath20, read_csv)
+```
+
+Create correct FIPS columns
+
+```{r}
+transport_tracts_2020 <- transport_tracts_2020 %>%
+  rename (GEOID = tract) %>%
+  mutate(
+    state = substr(GEOID, start = 2, stop = 3),
+    county = substr(GEOID, start = 4, stop = 6),
+    tract = substr(GEOID, start = 7, stop = 12)
+  )
+```
+
+Keep only variables of interest
+
+```{r}
+transit_trips_tracts_2020 <- transport_tracts_2020 %>%
+  select(GEOID, state, county, tract, blkgrps, population, households, transit_trips_80ami)
+
+transit_cost_tracts_2020 <- transport_tracts_2020 %>%
```

Comment:
This applies to all years - it would be great to have some test or viz here that confirms all files have been read in properly. Is there a count of tracts we should expect?